CHANGES.md

```diff
@@ -1,3 +1,17 @@
+## edge-19.11.3
+
+* CLI
+  * Added a check that ensures using `--namespace` and `--all-namespaces`
+    results in an error as they are mutually exclusive
+* Web UI
+  * Fixed a bug causing the `Run Linkerd Check` button to be incorrectly
+    positioned after being clicked
+* Internal
+  * Fixed an issue causing some control plane components to continue
+    using old certificates 
```

Comment:
Maybe you could be more specific here, saying that the tap, injector and sp-validator required to be be manually restarted when using Helm.